### DISCUSSION TITLE
Users who have read only permissions on a repository should be able to be a CODEOWNER of files in that repository.

### DISCUSSION THREAD

**Discussion by dat2** [community/community#8362] [Pull Requests]
In some use cases, we want non technical users (such as product managers) to have read only access to a repo. There are some monorepos with non adequate coverage of CODEOWNERS so we don't necessarily want those users to be able to create a PR and get it approved and merged. However, we also want those same users to be CODEOWNERS on a specific file in that repo. This would allow these users to block the PR and ensure the authors of those PRs contact these users for a review. For example, this might be enabling a feature in a certain code base and the user might want to delay that feature to be enabled until after a certain date.

We have considered creating a custom status check for this use case, but it's fairly high effort as a workaround.

**Comment by leofang** [comment]
NumPy is hitting the same issue. Is this not a support ticket that would get triaged? (Not sure how things work in this repo.)
